framework/src/executioners/SolveObject.C

```diff
@@ -13,11 +13,24 @@
 #include "FEProblem.h"
 #include "DisplacedProblem.h"
 
-SolveObject::SolveObject(Executioner & ex)
-  : MooseObject(ex.parameters()),
+registerMooseObject("MooseApp", SolveObject);
```

Comment:
Do you really want to add this as a registered object?

Reply:
Yes. `SolveObject` is a valid object performing empty operations. It can be a place holder for example as a replacement of `PicardSolve` when Picard iteration is not wanted without changing the code. Also if it replace `FEProblemSolve` we basically skip solving which I think it is better to let `FEProblem::solve` to skip the solving.

Reply:
That's pretty hacky.

Reply:
I can derive a solve object like `EmptySolve` which essentially has the `solve` function override like what we have in `SolveObject` in this PR. Do we need that?

Reply:
I don't know. You can set Picard max its to 1 for the first use case, and solve = false in Problem for the second one. So right now we don't. Maybe with the new system, setting solve=false in Problem might be awkward